src/api/treeTrackerApi.js

```diff
@@ -68,51 +71,75 @@ export default {
       handleError(error);
     }
   },
-  approveCaptureImage(id, morphology, age, captureApprovalTag, speciesId) {
+  approveCaptureImage(capture, morphology, speciesId, captureApprovalTag) {
     try {
-      const query = `${API_ROOT}/api/${getOrganization()}trees/${id}`;
+      log.debug(
+        'approveCaptureImage',
+        capture,
+        morphology,
+        speciesId,
+        captureApprovalTag
+      );
 
-      log.debug(query);
+      // map legacy data to fields for new microservice
+      const newCapture = {
+        id: capture.uuid,
+        reference_id: capture.id,
+        session_id: capture.session_id, // no legacy equivalent
+        grower_account_id: capture.planterId,
+        planting_organization_id: capture.organization_id,
+        device_configuration_id: capture.device_configuration_id, // no legacy equivalent
+        image_url: capture.imageUrl,
+        lat: capture.lat,
+        lon: capture.lon,
+        gps_accuracy: capture.gps_accuracy,
+        captured_at: capture.timeCreated,
+        note: capture.note,
+        morphology,
+        species_id: speciesId, // need uuid
```

Comment:
Users currently use Verify to add a species or some other tag to an already approved capture. We need to make sure we don't discard the existing `capture.speciesId` if one exists and the user hasn't set another (this highlights the need to make the UX clearer in this use case).